lib/update-state.js

```diff
@@ -30,6 +29,11 @@ function updateState (state) {
 
 		var currentState = state[version];
 
+		if (currentState === states.preparing) {
+			icon = chalk.grey(figures.circleDotted);
+			message = chalk.grey('preparing');
```

Comment:
"preparing" does not clearly explain what is going on, "starting services" would be better.